stable/admin/templates/role.yaml

```diff
@@ -34,11 +34,13 @@ rules:
   - create
   - update
 {{- if eq (include "defaulttrue" .Values.nuodb.addClusterRoleBinding) "true" }}
+{{- $namespace := default .Release.Namespace .Values.admin.namespace -}}
```

Comment:
Trim the namespace so that when combined with the release name and `nuodb-kube-inspector` suffix, it is not too huge to be a legal resource name. E.g.

```
{{- $namespace := default .Release.Namespace .Values.admin.namespace | trunc 50 | trimSuffix "-" -}}
```

Reply:
Why trim just the namespace and not the release name? It seems better to just truncate the full resource name.

Reply:
The release name is already trimmed by `admin.fullname`.

Reply:
1. It doesn't seem that kubernetes cares about lengths here. I was able to create a ClusterRole and ClusterRoleBinding with a 286 character long name.
2. If we want to keep names to under 63 characters, we can't just append a 50 character namespace to a 50 character full name without trimming. Maybe feed all of it to `truncWithHash` so that very long names are still unique?